client/src/app/pages/main-feeds/post/Post.scss

```diff
@@ -0,0 +1,90 @@
+.container {
+	background-color: white;
+	width: 100%;
+	border-radius: 10px;
+	padding-top: 13px;
+	margin-top: 15px;
+	box-shadow: 0px 5px 8px -1px rgba(0, 0, 0, 0.75);
+
+	img {
```

Comment:
should not use this class because other img in other component will be conflicted